pkgs/by-name/sc/sccmhunter/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "sccmhunter";
+  version = "1.0.6-unstable-2024-10-30";
+  pythonRelaxDeps = true;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "garrettfoster13";
+    repo = "sccmhunter";
+    rev = "7f493d1499fec1a694d949d9de17f25acdb6a0a9";
+    hash = "sha256-rMkm4SDIiPrlNJWlnwD2td7yzkTNlTzZ3KaiKK5TOfg=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    cmd2
+    cryptography
+    impacket
+    ldap3
+    pandas
+    pyasn1
+    pyasn1-modules
+    requests
+    requests-kerberos
+    requests-ntlm
+    requests-toolbelt
+    rich
+    tabulate
+    typer
+    urllib3
+    pyopenssl
+    pycryptodome
+  ];
+
+  meta = with lib; {
+    description = "Post exploitation tool to identify and attack SCCM related assets in an Active Directory domain";
+    homepage = "https://github.com/garrettfoster13/sccmhunter";
+    license = licenses.mit;
+    mainProgram = "sccmhunter.py";
+    maintainers = with maintainers; [ purpole ];
+  };
```

```suggestion
  meta = {
    description = "Post exploitation tool to identify and attack SCCM related assets in an Active Directory domain";
    homepage = "https://github.com/garrettfoster13/sccmhunter";
    changelog = "https://github.com/garrettfoster13/sccmhunter/blob/${src.rev}/changelog.md";
    license = lib.licenses.mit;
    mainProgram = "sccmhunter.py";
    maintainers = with lib.maintainers; [ purpole ];
  };
```

Please avoid `with lib` here. It may cause a confusion. refer: https://github.com/NixOS/nixpkgs/issues/208242